src/components/select/select.tsx

```diff
@@ -225,30 +253,14 @@ export class Select {
         this.menuOpen = false;
     }
 
-    private handleMenuKeyDown(event: KeyboardEvent) {
+    private handleMenuTriggerKeyPress(event: KeyboardEvent) {
         const isEnter = event.key === ENTER || event.keyCode === ENTER_KEY_CODE;
         const isSpace = event.key === SPACE || event.keyCode === SPACE_KEY_CODE;
 
-        if (isSpace || isEnter) {
+        if (!this.menuOpen && (isSpace || isEnter)) {
```

Comment:
It's great that this doesn't open automatically when tabbed to. We should make the object-picker work this way as well. Currently, when there's nothing selected in an object-picker, you have to tab through the whole list of suggestions, and it's impossible to pass an object-picker when tabbing backwards. This component is much more well behaved! 👍👍